pandas_dev_flaker/_plugins_tree/conftest.py

```diff
@@ -12,7 +12,7 @@ def visit_ImportFrom(
     node: ast.ImportFrom,
     parent: ast.AST,
 ) -> Iterator[Tuple[int, int, str]]:
-    if node.module == "conftest":
+    if isinstance(node.module, str) and "conftest" in node.module:
```

Comment:
Cool, thanks for fixing and adding a test! Would it work to, instead, check if `conftest` is in `state.from_imports['pandas']`?

Reply:
ooo let me check! sorry i didnt know how "fuzzy" the match should be

Reply:
actually, couldn't resist to check it now :)

I presume you did the `isinstance` check to satisfy the `mypy` check? If so, I think we just need to check it's not `None`. And the second part, I think I'd split on `'.'` and check the last element, so something like:
```python
    if node.module is not None and node.module.split('.')[-1] == 'conftest':
```

Reply:
yes i did heh. oooo nice, looks like that works 🎊

Reply:
updated!